cowprotocol/cowswap
# feat(trezor): display snackbar on wallet account changes

# Summary

Added a snackbar notification on account changes for hardware wallets.

<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/52f6a890-e9cc-4e33-9164-cbee0992002c">

<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/412883ed-5769-45b9-a2e0-6d1b4f9154a6">

<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/bbe64a46-9358-4427-9e48-82b875c51774">

<img width="400" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/0539be53-dec8-4e5f-b1b8-39633e740def">

## To test
1. Connect to hardware wallet (Trezor)
2. Open Account modal
3. Click "Change Account"
4. Change account and click "Connect selected account"
5. "Select account" should be closed
6. Snackbar with text "Trezor account changed" should be displayed

<img width="600" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/549bcba4-828e-4b5d-8848-de76d3824a8c">
